src/definitions.ts

```diff
@@ -58,28 +58,38 @@ export abstract class ExportsWalker {
 
   /** Walks all elements and calls the respective handlers. */
   walk(): void {
-    for (let file of this.program.filesByName.values()) {
+    // for (let file of this.program.filesByName.values()) {
```

Comment:
Could you add TODO comment about this temp solution and in future will revert back to `for of`?